sematic/types/types/image.py

```diff
@@ -2,8 +2,25 @@
 import hashlib
 from dataclasses import dataclass
 
-# Third-party
-import magic  # type: ignore
+try:
+    # Third-party
+    import magic  # type: ignore
+except ImportError as e:
+
+    # Standard Library
+    import platform
+
+    template = "Sematic requires the `libmagic` package. Please install it{os_message}"
+    mapping = {
+        "Darwin": " with `brew install libmagic`, or with `port install file`",
+        "Linux": (
+            " with `sudo apt-get install libmagic1`, "
+            "or with `sudo yum install file-devel`"
+        ),
+    }
+
+    error_message = template.format(os_message=mapping.get(platform.system(), "."))
+    raise ImportError(error_message) from e
```

Comment:
Can we maybe turn this into a warning at import time, and then fail at runtime with the same error message if somebody tries to instantiate the `Image` type? Why? Because for somebody that has just `pip installed` us to play around, we want to minimize friction of extra things they have to install to get something working. It would be great if we could preserve the experience that after a `pip install`, you can start playing right away. That also mirrors our behavior with pandas/torch types.

Reply:
Excellent suggestion! Done.